skeleton/backstage/template.yaml

```diff
@@ -90,6 +90,28 @@ spec:
                     - Jenkins (SLSA 2)
                     - Azure Pipelines (SLSA 2)
                     - Github Actions (SLSA 2)
+              dependencies:
+                ciType:
+                  oneOf:
+                    - properties:
+                        ciType:
+                          const: tekton
+                    - properties:
+                        ciType:
+                          const: jenkins
+                    - required: 
+                        - azureProject
+                        - ciType
+                      properties:
+                        ciType:
+                          const: azure
+                        azureProject:
+                          title: Azure Project
+                          type: string
+                          ui:help: The project the run the pipelines under
```

Comment:
"to run" ?

Reply:
Oops fixed that grammar error under https://github.com/redhat-appstudio/tssc-sample-templates/pull/121/commits/87dfa3bb986d46aba58ea8bf66d0571b7e782d14.